avivator/src/components/Viewer.jsx

```diff
@@ -85,6 +91,7 @@ const Viewer = () => {
       isLensOn={isLensOn}
       onViewportLoad={onViewportLoad}
       extensions={[new LensExtension()]}
+      onViewStateChange={onViewStateChange}
```

Comment:
```suggestion
```
I think we either need to update the footer or not do this for side by side.

Reply:
changed.